core/templates/external-js/tiddlywiki5-external-js.html.tid

```diff
@@ -37,5 +37,5 @@ title: $:/core/templates/tiddlywiki5-external-js.html
 <!--~~ Ordinary tiddlers ~~-->
 {{$:/core/templates/store.area.template.html}}
 </body>
-<script src="%24%3A%2Fcore%2Ftemplates%2Ftiddlywiki5.js" onerror="alert('Error: Cannot load tiddlywiki.js');"></script>
+<script src="tiddlywiki.js" onerror="alert('Error: Cannot load tiddlywiki.js');"></script>
```

Comment:
This change breaks the web server usage. Perhaps we need to make the path to the script be a retained in a configuration tiddler? Alternately, we need to parameterise this template so that we can pass the script path as a variable, and then create two new wrapper templates that set the variable and invoke this template.